generators/pythonv2/codegen/src/ast/__test__/__snapshots__/Class.test.ts.snap

```diff
@@ -37,3 +37,11 @@ exports[`class > inherits from two parent classes 1`] = `
     pass
 "
 `;
+
+exports[`class > should generate a class with local classes 1`] = `
+"class OuterClass:
+    class InnerClass(ParentClass):
```

Comment:
[q] where does `ParentClass` come from? Are we missing an import statement, or is that oos for this PR?